sdk/scheduler/src/test/resources/valid-envKey-ports.yml

```diff
@@ -0,0 +1,26 @@
+name: "data-store"
+pods:
+  meta-data:
+    count: 2
+    tasks:
+      meta-data-task:
+        goal: RUNNING
+        cmd: "echo $TASK_NAME >> $TASK_NAME$CONTAINER_PATH_SUFFIX/output && sleep $SLEEP_DURATION"
+        cpus: 0.1
+        memory: 512
+        ports:
+          http:
+            port: 8080
+            env-key: some-key
+          no-key:
+            port: 8088
+          key-key:
+             port: 8089
+        volume:
+          path: "meta-data-container-path"
+          type: ROOT
+          size: 5000
+        env:
+          TASK_NAME: "meta-data"
+          CONTAINER_PATH_SUFFIX: "-container-path"
```

Comment:
Why this? It's just a test YAML, I think it's more readable if the path is just there in the command.